helix-lsp/src/lib.rs

```diff
@@ -359,7 +359,7 @@ impl Registry {
                 let (_, old_client) = entry.insert((id, client.clone()));
 
                 tokio::spawn(async move {
-                    if let Err(e) = old_client.shutdown_and_exit().await {
+                    if let Err(e) = old_client.force_shutdown().await {
```

Comment:
If we're opting for this instead, this method already prints an error message on shutdown failure, so we don't need one here too. It should suffice to just call the method.